easybuild/easyblocks/generic/scons.py

```diff
@@ -54,7 +54,8 @@ def test_step(self):
         Test with SCons 
         """
         cmd = "scons check"
-        (out, _) = run_cmd(cmd, log_all=True)
+        if self.cfg['runtest']:
+            (out, _) = run_cmd(cmd, log_all=True)
```

Comment:
As far as I understand, `check` is just a target like in a Makefile. It can have another name, it can take arguments. So, why not add the flexibility?